hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -121,13 +124,16 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
             .setVolumeName(volumeName).setBucketName(bucketName);
 
     boolean deleteStatus = true;
+    long startNanos = Time.monotonicNowNanos();
     try {
+      long startNanosDeleteKeysResolveBucketLatency = Time.monotonicNowNanos();
       ResolvedBucket bucket =
```

Comment:
Here as well, the [`MetricUtil#captureLatencyNs`](https://github.com/apache/ozone/blob/master/hadoop-hdds/common/src/main/java/org/apache/hadoop/util/MetricUtil.java) block can directly be wrapped around the `resolveBucketLink` call. This should make the code less verbose and more readable.

Reply:
In resolveBucketLink we have Volume and Bucket as a local variable but to use captureLatencyNs we need to make them final. So without making those variable as final we can't use them here. That's why to keep it simple I captured it in another way, i.e. capturing start and end time.